packages/cli/src/commands/dev.ts

```diff
@@ -107,16 +108,28 @@ export async function dev({
     ? toolPathsWithFileNames.map(tool => path.join(dotMastraPath, 'tools', `${tool.name}.mjs`)).join(',')
     : undefined;
 
-  const proc = execa('node', ['server.mjs'], {
-    cwd: dotMastraPath,
-    env: {
-      port: `${port} || 4111`,
-      MASTRA_TOOLS_PATH,
-    },
-  });
+  try {
+    console.log('Starting server...');
```

Comment:
we can remove the stray logs now i think right?

Reply:
Not sure, I think it's useful for the user to know when the server starts